apps/bigclock/big_clock.star

```diff
@@ -150,8 +162,18 @@ def main(config):
     is_24_hour_format = config.bool("is_24_hour_format", DEFAULT_IS_24_HOUR_FORMAT)
     has_leading_zero = config.bool("has_leading_zero", DEFAULT_HAS_LEADING_ZERO)
     has_flashing_seperator = config.bool("has_flashing_seperator", DEFAULT_HAS_FLASHING_SEPERATOR)
-    color_daytime = config.get("color_daytime", DEFAULT_COLOR_DAYTIME)
```

Comment:
A bit over half of all installations of `big-clock` are using a non-default value for `color_daytime` or `color_nighttime`. It would be great if we could make this change backwards compatible so that these installations continue to function as before.

Reply:
I was also a bit concerned about backwards compatibility, and I agree that most users will find the RGB values confusing. I was actually considering using HSL instead, as that's a little easier to understand, but ultimately against it.

I could not think of a way to support both the selector and the default color, unless the user types in a hex value — but again, that would require input validation.

I believe the best solution here is a color picker schema.

Reply:
Color picker schema would be great. It's just a matter of finding the time to build it out. Lot of other priorities as always. =(

For backwards compatibility, the app should still receive the `color_daytime` parameter for existing installations (even when it's been removed from `get_schema()`), so it'd just be a matter of checking for that field first (with the same default) and then letting the new RGB parameters override it.

Reply:
Yes, it should still be passed the same way for older installations.

When there's a colorpicker field, you'd need to introduce a new field ID, like `colors_daytime_rgb`, to keep backwards compatibility.

Reply:
Since the RGB values are not intuitive, and there would be more complexity switching back and forth between the various methods to achieve backwards compatibility, I think it would be best not to use the RBG color selectors for now, and wait until there is an official colorpicker schema.

I'm going to close the pull request, and hopefully we can come back to this at a later date. Thanks!